odxtools/ecu_variant_matcher.py

```diff
@@ -97,7 +97,7 @@ def decode_ident_response(
                 resolved for any positive or negative response."
         )
 
-    def __init__(self, ecu_variant_candidates: List[DiagLayer], use_cache: bool = True):
```

Comment:
Not an XML model class, I think the default can stay

Reply:
ok